components/quizSingle/ResultView.tsx

```diff
@@ -0,0 +1,70 @@
+import Link from "next/link";
+import { QuizRecord } from "~/models/ChinguQuiz/QuizRecord";
+import { Heading4, TextBodyBold, TextBodySmallBold } from "../shared/styles";
+import {
+  ResultPageContainer,
+  ResultTitleContainer,
+  ResultTileContainer,
+  ResultTile,
+  CodeBlock,
+  TileNumber,
+  SubmitQuizBtnStyled,
+} from "./styles";
+import ScoreGraph from "./ScoreGraph";
+
+export default function ResultView({
+  quizTitle,
+  quizRecord,
+}: {
+  quizTitle: string;
+  quizRecord: QuizRecord[];
+}) {
+  let percentage = Math.round(
```

Comment:
Only use let if the value is going to change. Since these values won't change, they should be declared as const

Reply:
Good catch, fixed.